Makefile

```diff
@@ -29,6 +29,7 @@
 #   - clean-all - cleans the build area and release packages
 #   - docker-thirdparty - pulls thirdparty images (postgres)
 #   - gotools - Installs go tools, such as: golint, goimports, gocov
+#	- vendor - vendors third-party packages
```

Comment:
nit: this has a tab instead of spaces; nbd but, consistency

Reply:
done